apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -58,7 +58,7 @@ const requiredFields = ['gradesAtSchool', 'firstName', 'lastName', 'primaryEmail
   'principalLastName', 'principalPrefix', 'principalEmail', 'selectedCourse'];
 const requiredCsdFields = ['gradesPlanningToTeach'];
 const requiredCspFields = ['cspDuration', 'cspApCourse', 'gradesPlanningToTeach', 'cspApExamIntent'];
-const requiredSurveyFields = ['committedToSummer', 'ableToAttendAssignedSummerWorkshop', 'allStudentsShouldLearn',
```

Comment:
is this no longer a required field? If not, also remove from the [model](https://github.com/code-dot-org/code-dot-org/blob/1ac91343916039a884c1895439174fd82ba55836/dashboard/app/models/pd/teacher_application.rb#L54-L54)